src/Rokt-Kit.js

```diff
@@ -153,6 +154,15 @@ var constructor = function () {
         return data;
     }
 
+    function sanitizeIdentities(_data) {
```

Comment:
I think `sanitize` is too vague and generic.

```suggestion
    function sanitizeEmailIdentities(_data) {
```